arm/compute/client.go

```diff
@@ -1,3 +1,7 @@
+// Package compute implements the Azure ARM Compute service API version
+// 2015-06-15.
+//
+// The Compute Management Client..
```

Comment:
it's actually not a big deal, we can merge as the first line is what's going to be shown at https://godoc.org/github.com/Azure/azure-sdk-for-go/arm